src/Build/Microsoft.Build.csproj

```diff
@@ -234,6 +233,7 @@
     <Compile Include="BackEnd\Components\SdkResolution\ISdkResolverService.cs" />
     <Compile Include="BackEnd\Components\SdkResolution\OutOfProcNodeSdkResolverService.cs" />
     <Compile Include="BackEnd\Components\SdkResolution\OutOfProcNodeSdkResolverServiceFactory.cs" />
+    <Compile Include="BackEnd\Components\SdkResolution\SdkResolverManifest.cs" />
```

Comment:
extreme nit: move this down so it's alphabetical? But alphabetical order is a mess in this file anyway so meh.